pygmt/tests/test_datasets_samples.py

```diff
@@ -186,3 +186,20 @@ def test_maunaloa_co2():
     assert summary.loc["max", "date"] == 2019.3699
     assert summary.loc["min", "co2_ppm"] == 313.2
     assert summary.loc["max", "co2_ppm"] == 414.83
+
+
+def test_rock_sample_compositions():
+    """
+    Check that the @ternary.txt dataset loads without errors.
+    """
+    data = load_sample_data(name="rock_compositions")
+    assert data.shape == (1000, 4)
+    summary = data.describe()
```

Comment:
I saw that codes like `assert summary.loc["min", "water"] == 0` are used here and also other datasets. Why don't we use codes like `assert data["water"].min() == 0`?

Reply:
Not sure, just oriented on the other tests. However, I can adjust it, also for the other tests. Maybe in another PR @seisman ?

Reply:
Sounds good